service/src/main/resources/changelog/changesets/20230126_sarscov2_nextstrain_io_updates.yaml

```diff
@@ -0,0 +1,1485 @@
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.dedup_seqs.docker",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.dedup_seqs.error_on_seq_diff",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "Boolean"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.dedup_seqs.nextstrain_ncov_repo_commit",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.derived_cols.docker",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.derived_cols.lab_highlight_loc",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.derived_cols.table_map",
```

Comment:
The UI is showing this as "undefined(optional)", which we might want to dig into. I suspect it's a UI display issue rather than anything wrong in prestaging, just something to be aware of

Reply:
ah, so it is... interesting.

I wonder if it's worth holding off on merging this until we know for sure that it's just a UI display issue